antsibull/data/sphinx_init/build_sh.j2

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e
+cd @{ dest_dir }@
+
+# Create collection documentation into temporary directory
+rm -rf temp-rst
+mkdir -p temp-rst
+{% if use_current %}
+{%   if collections | length > 0 %}
+antsibull-docs collection \
+    --use-current \
+    --dest-dir temp-rst \
+{%     if squash_hierarchy %}
+    --squash-hierarchy \
+{%     endif %}
+    @{ ' '.join(collections) }@
+{%   else %}
+antsibull-docs current --dest-dir temp-rst
+{%   endif %}
+{% else %}
+antsibull-docs collection \
+{%     if collection_version != '@latest' %}
+    --collection-version @{ collection_version }@ \
+{%     endif %}
+    --dest-dir temp-rst \
+{%     if squash_hierarchy %}
+    --squash-hierarchy \
+{%     endif %}
+    @{ ' '.join(collections) }@
+{% endif %}
+
+# Copy collection documentation into source directory
+rsync -avc --delete-after temp-rst/collections/ rst/collections/
+
+# Build Sphinx site
+sphinx-build -M html rst build -c .
```

Comment:
Make sure to not tolerate any broken references:
```suggestion
sphinx-build -W --keep-going -M html rst build -c .
```

Reply:
Improved in 2421c18cb71424fc43fc9b1f5cfb93666a2c9b9b. I had to move the arguments to the end, to avoid errors such as:
```
usage: sphinx-build [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]
sphinx-build: error: unrecognized arguments: -M
```